### PR TITLE
systemctl: fix token syntax for multicommand examples

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/systemctl.md [hunk 1]
```diff
@@ -9,19 +9,19 @@
...
 - Start/Stop/Restart/Reload a service:
 
-`systemctl start/stop/restart/reload {{unit}}`
+`systemctl {{start|stop|restart|reload}} {{unit}}`
 
 - Show the status of a unit:
 
 `systemctl status {{unit}}`
 
 - Enable/Disable a unit to be started on bootup:
```
Comment (bl-ue) [inline pages/linux/systemctl.md:18]:
```suggestion
- Enable/disable a unit to be started on bootup:
```

File: pages/linux/systemctl.md [hunk 2]
```diff
@@ -9,19 +9,19 @@
 
 - Start/Stop/Restart/Reload a service:
```
Comment (bl-ue) [inline pages/linux/systemctl.md:10]:
```suggestion
- Start/stop/restart/reload a service:
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

**Review by bl-ue — APPROVED** [review]

File: pages/linux/systemctl.md [hunk 3]
```diff
@@ -7,21 +7,21 @@
...
 
-`systemctl start/stop/restart/reload {{unit}}`
+`systemctl {{start|stop|restart|reload}} {{unit}}`
 
 - Show the status of a unit:
 
 `systemctl status {{unit}}`
 
-- Enable/Disable a unit to be started on bootup:
+- Enable/disable a unit to be started on bootup:
```
Comment (sbrl) [inline pages/linux/systemctl.md:18]: Don't think this really needs changing? It's fine as-is though.

**Review by sbrl — APPROVED** [review]
Thanks, @navarroaxel!

File: pages/linux/systemctl.md (hunk 3 above)
Comment (bl-ue) [reply pages/linux/systemctl.md:18]: Oh you don't?  It seemed to me incorrect.  If you think so we can revert it.

File: pages/linux/systemctl.md [hunk 4]
```diff
@@ -9,19 +9,19 @@
...
 - Show the status of a unit:
 
 `systemctl status {{unit}}`
 
 - Enable/Disable a unit to be started on bootup:
 
-`systemctl enable/disable {{unit}}`
+`systemctl {{enable|disable}} {{unit}}`
 
 - Mask/unmask a unit to prevent enablement and manual activation:
```
Comment (bl-ue) [inline pages/linux/systemctl.md:22]: Ooops this should be reverted too @navarroaxel (`unmask` → `Unmask`).  We did that in another PR but...

Comment (navarroaxel) [reply pages/linux/systemctl.md:22]: Done https://github.com/tldr-pages/tldr/pull/5281

**Pull request merged**